### ISSUE TITLE
ActionText and Globalize error

### ISSUE THREAD

**Issue by alec-c4** [globalize/globalize#731]
Hi!
I have application which is using globalize gem and i planning to start using ActionText. So, for example i have model called Business

```
class Business < ApplicationRecord
  translates :name, :description
  globalize_accessors

  has_rich_text :description
end 
```

and i created an record in database. But on attempt to edit i see following error

```
undefined method `body' for "<div><strong>jfgjwhgewr</strong></div>":String
```

for my form

```
  .form-inputs
    - I18n.available_locales.each do |locale|
      = f.input :"name_#{locale.to_s}", label: "Name (#{locale.to_s})"
    - I18n.available_locales.each do |locale|
      = f.label "Description (#{locale.to_s})"
      = f.rich_text_area :"description_#{locale.to_s}"
```
Whats wrong with it and how can i solve this issue?

PS: I found https://github.com/rails/actiontext/issues/32#issuecomment-450653800 but this solution doesn't work :(

**Comment by balakirevs** [comment]
may be try to use the following in the view:

```
<%= rich_text_area_tag "business[name]",  f.object.name %>
```

**Comment by fareesh** [comment]
The solution listed here: https://github.com/rails/actiontext/issues/32#issuecomment-450653800 seems to work fine for me, however it requires at least one non-actiontext field in your model to also be translated. In my case I had only one field in my model which required translation - i.e. the actiontext field, so I had to make a dummy field in which I populated the current time and it worked flawlessly.
